buildpacks/nodejs-pnpm-install/src/main.rs

```diff
@@ -142,6 +145,15 @@ impl Buildpack for PnpmInstallBuildpack {
     }
 }
 
+fn disable_pnpm_auto_update_notifier(env: &Env) {
```

Comment:
I think this change should be in its own PR? This PR has the "skip changelog" label applied, and only mentions updating test snapshots, whereas this addition is a user-facing behaviour change?

Reply:
Sorry, I should have switched this PR to draft after noticing the update notifiers. I wasn't sure if this was what I wanted to do but I was having trouble with getting a filter that I was happy with 😅 

This has been reverted in https://github.com/heroku/buildpacks-nodejs/pull/1082/commits/bb2d25ddeb3e8a2451d0a7d61593a7623bdd64f6